src/main/java/com/keshavarzi/airplanestats/model/UserEntity.java

```diff
@@ -0,0 +1,141 @@
+package com.keshavarzi.airplanestats.model;
+
+import jakarta.annotation.Nonnull;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.ForeignKey;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Collection;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "user", schema = "user_data")
+public class UserEntity implements Cloneable {
+
+    @Id
+    @Nonnull
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", table = "user", unique = true, nullable = false)
+    private Long userId;
+
+    @Nonnull
+    @Column(name = "email", table = "user", unique = true, nullable = false)
+    private String email;
+
+    @Nonnull
+    @Column(name = "password", table = "user", nullable = false)
+    private String password;
```

Comment:
Note: A password should ***not*** be stored in plain text. It should be stored encrypted or better; hashed.